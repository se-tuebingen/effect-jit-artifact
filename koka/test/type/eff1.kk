fun omega(): div () {
  return ()
}

fun main() {
  fun choose(x,y){ if (True) then x else y}
  choose(throw("hi"),omega())
}