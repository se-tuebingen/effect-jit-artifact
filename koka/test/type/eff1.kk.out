type/eff1/main: () -> pure ()
type/eff1/omega: () -> div ()